### DISCUSSION TITLE
Are hierarchical entities created when glTF scenes are imported?

### DISCUSSION THREAD

**Discussion by vadimb892** [bevyengine/bevy#12831] [Q&A]
I want to use Parent, Children queries on entities from glTF file. But results of these queries are empty. Are hierarchical entities created when glTF scenes are imported? If not, how can these relationships be recreated based on GltfNode?

**Comment by rcorre** [comment]
Bevy does create a hierarchy when loading a GLTF scene. It creates a top-level node for the scene [here](https://github.com/bevyengine/bevy/blob/main/crates/bevy_gltf/src/loader.rs#L598), then a child entity for each child node [here](https://github.com/bevyengine/bevy/blob/main/crates/bevy_gltf/src/loader.rs#L1253-L1254). The [skinned mesh](https://github.com/bevyengine/bevy/blob/main/crates/bevy_gltf/src/loader.rs#L1253-L1254) example demonstrates how to walk the hierarchy at runtime.
